### PR TITLE
Update homepage testimonial section

### LINKED ISSUE #1577: Update the homepage testimonial section with new testimonials
**Is your feature request related to a problem? Please describe.**
The home page includes old testimonials.

**Describe the solution you'd like**
Update the testimonial section with new testimonials

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose

The purpose of this PR is to fix #1577

## Goals
To update the home page testimonial section
## Approach
Updated the home page testimonials & card styles

### Screenshots
image  

### Preview Link

https://pr-1578-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by gajindubandara** [comment]
Note -
1. Ramith and Nayanathara's designations have not yet been added. (I'll update it)
2. There is a CSS section used for the old testimonial styles in the `global-custom` file. Previously it was used for the home page testimonials and the upspeak page testimonial, shall we remove that old CSS section and update the upspeak page testimonial the same as the new home page testimonial section

WDYT?? @anjula-sack

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
@gajindubandara this should be the preview link right? 
- https://pr-1578-sef-site.surge.sh/

Can you change it?

**Comment by gajindubandara** [comment]
> @gajindubandara this should be the preview link right? 
> - https://pr-1578-sef-site.surge.sh/
> 
> Can you change it?

Done!

File: index.html [hunk 1]
```diff
@@ -435,69 +435,162 @@ <h1 class="display-3 text-center mb-5">Infrastructure partner</h1>
...
+                                    Testimonials in Empowering Aspiring Entrepreneurs to Overcome Challenges, Pursue
+                                    Their Dreams, and Foster a Supportive Community."
+                                </div>
                             </div>
                         </div>
+                        <p class="paragraph text-justify">
+                            It's great to work with your team and I wish you good luck. It's a great opportunity for me to
+                            share my stories with upcoming entrepreneurs and I believe you will keep your great service
+                            to motivate the rest.
+                        </p>
```
Comment (anjula-sack) [inline index.html]: Let's remove Sanmugathasan Prasanthan from here @gajindubandara

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -435,69 +435,162 @@ <h1 class="display-3 text-center mb-5">Infrastructure partner</h1>
...
-                                <h5 class="name">Vladimir Kovacevic</h5>
-                                <p class="uni">Bioinformatics Analyst, Seven Bridges Genomics</p>
+            <div id="card_container">
+                <div class="row testimonialBox shadow">
+                    <div class="col-md-12">
+                        <div class="row">
+                            <div class="col-md-3">
+                                <div class="mentee col">
+                                    <div class="row authorDetails">
+
```
Comment (YoshithaRathnayake) [inline index.html]: Is this line required? @gajindubandara 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/global-custom.css [hunk 3]
```diff
@@ -772,3 +772,70 @@ body {
...
+}
+
+.authorImg .img {
+    border-radius: 50% !important;
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    border: 5px solid #EAECEE;
+}
+
```
Comment (YoshithaRathnayake) [inline assets/css/global-custom.css]: Is this line also required? @gajindubandara 
If not needed,
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/global-custom.css (hunk 3 above)
Comment (YoshithaRathnayake) [reply assets/css/global-custom.css]: Oh sorry @gajindubandara we need that line. That is the end of the line, please add it again, sorry for the inconvenience

image

File: assets/css/global-custom.css [hunk 4]
```diff
@@ -772,3 +772,69 @@ body {
...
+    margin-bottom: 2px;
+}
+
+.authorImg .img {
+    border-radius: 50% !important;
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    border: 5px solid #EAECEE;
+}
```
Comment (YoshithaRathnayake) [inline assets/css/global-custom.css]: - Sorry @gajindubandara we need that line. That is the end of the line, please add it again, sorry for the inconvenience.
- Just commit the changes, it'll be easier for you

```suggestion
}

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/global-custom.css (hunk 3 above)
Comment (YoshithaRathnayake) [reply assets/css/global-custom.css]: See below I have suggested the changes. Commit them @gajindubandara

**Pull request closed without merging**